runelite-client/src/main/java/net/runelite/client/plugins/friendnotes/FriendNotesPlugin.java

```diff
@@ -152,16 +151,16 @@ public void onMenuEntryAdded(MenuEntryAdded event)
 		final int groupId = WidgetInfo.TO_GROUP(event.getActionParam1());
 
 		// Look for "Message" on friends list
-		if (groupId == WidgetInfo.FRIENDS_LIST.getGroupId() && event.getOption().equals("Message"))
+		if (groupId == WidgetInfo.FRIENDS_LIST.getGroupId() && event.getOption().equals(MESSAGE))
```

Comment:
The "Message" string shouldn't need a static field if it's only used here. I'd leave it as a string literal.